interface/themes/core/navmenu.scss

```diff
@@ -62,7 +62,7 @@ body {
 /* $GLOBALS['style']['BGCOLOR2'] */
 
 .body_bottom {
-    background: url('../pic/aquabg.gif') repeat;
+    background: url('interface/pic/aquabg.gif') repeat;
```

Comment:
Is this testing to work?
Seems like it should be:
```
background: url('../../interface/pic/aquabg.gif') repeat;
```

Reply:
I agreed with @bradymiller

Reply:
Hi @bradymiller @stu01509 ,
`background: url('../../interface/pic/aquabg.gif') repeat;`
gives the same error.
`background: url('interface/pic/aquabg.gif') repeat;`
works fine.

Reply:
Hi @harshbawari 

Try this
`background: url('./../../../interface/pic/aquabg.gif') repeat;`

Reply:
Hi guys, I think you need to remember that this include is a relative path. So it will be relative to where it is called and not where it is imported on theme build. ie the file doesn't get imported but the path does.
So if you adjust this path to work in one place doesn't mean it will in another.
make sense?